Sources/FluentKit/Query/Builder/QueryBuilder.swift

```diff
@@ -11,12 +11,14 @@ public final class QueryBuilder<Model>
     internal var shouldForceDelete: Bool
     internal var models: [Schema.Type]
     public var eagerLoaders: [AnyEagerLoader]
+    public var history: QueryHistory
 
-    public convenience init(database: Database) {
+    public convenience init(database: Database, history: QueryHistory = .init()) {
         self.init(
             query: .init(schema: Model.schema),
             database: database,
-            models: [Model.self]
+            models: [Model.self],
+            history: history
```

Comment:
Shouldn't this use `database.context.history`?

Reply:
Actually should be `database.history` since `database.logger`, `database.eventLoop`, etc extensions exist for context properties.